tests/tests_default.yml

```diff
@@ -2,3 +2,28 @@
 - hosts: all
   roles:
     - linux-system-roles.cockpit
+
+  tasks:
+    - name: "test: socket is active"
+      command: systemctl is-active cockpit.socket  # noqa 301 303 command-instead-of-module no-changed-when
+      register: result
+      failed_when: result.failed
```

Comment:
I think you can remove the `register` and `failed_when` here.  You can get rid of the `noqa no-changed-when` by adding `changed_when: false`
If you don't use a `:` in the value of the `name:`, you don't need to use double quotes e.g.
`name: test - socket is active`

Reply:
Indeed, thanks! I cleaned this up as you suggested, much better.